nix-on-droid/default.nix

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) 2019-2020, see AUTHORS. Licensed under MIT License, see LICENSE.
 
-{ bash, coreutils, nix, runCommand }:
+{ bash, coreutils, nix, nix_2_4, runCommand }:
```

Comment:
Since we are in release 21.11, we should be able to assume nixpkgs 21.11. Therefore using the stable nix 2.4 release instead of an moving pointer to a nix release that supports flakes felt more reliable.

Reply:
Fine as long as we don't forget to unpin it next time, I guess. And I hope next time we'll revisit it anyway because we'll have flakes in stable.

Reply:
I think for 21.05 nix 2.4 will become the default in nixpkgs and flakes should be marked as stable. After that we will remove that here.